src/ActionList/Item.tsx

```diff
@@ -358,6 +357,8 @@ export function Item(itemProps: Partial<ItemProps> & {item?: ItemInput}): JSX.El
       variant={variant}
       showDivider={showDivider}
       aria-selected={selected}
+      aria-labelledby={text ? `${id}-label` : undefined}
```

Comment:
Does this override the default speech for the checkboxes in SelectPanel?  If not, we may want to consider removing the `aria-label` on the `MultiSelectInput` below, which I believe is what causes the doubling effect we noticed last week.

Reply:
Great question! I’d like to follow-up on this in a separate issue/PR. Earlier this week, @ichelsea, @JoseInTheArena, and I discussed using the [`menuitemradio`](https://www.w3.org/TR/wai-aria-1.2/#menuitemradio) and [`menuitemcheckbox`](https://www.w3.org/TR/wai-aria-1.2/#menuitemcheckbox) roles for single-select and multi-select `Item`s, respectively.

Demonstrations of this pattern in WAI-ARIA Authoring Practices 1.2 such as [“Editor Menubar Example”](https://w3c.github.io/aria-practices/examples/menubar/menubar-editor.html) do not include separate `<input type="checkbox">`, so we might want to remove `<MultiSelectInput>` from the Accessibility Tree (e.g. using `aria-hidden="true"`; ref [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute#deciding_between_aria-hiddentrue_rolepresentation_and_rolenone)). Similarly, [“Example 4”](https://www.w3.org/TR/wai-aria-1.2/#example-4) includes a purely decorative checkbox image with `role="presentation"` inside a `menuitemcheckbox`.

Reply:
Opened https://github.com/github/primer/issues/228 to track this

Reply:
Sounds like a great plan, thanks for opening an issue to track it!